addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -806,6 +875,10 @@ export class SelectionPlugin extends Plugin {
         return [node, offset];
     }
 
+    isNodeEditable(node) {
+        return node.parentElement?.isContentEditable;
+    }
```

Comment:
The name of this function is confusing IMO. I originally did not realize it was exported as an `editable_nodes_predicates` so I thought this was some local function that the `selection_plugin` was using to determine editability. Considering it's extremely short I would have inlined it in the resources to minimize any confusion.